frontend/src/services/ChartDataFormatter.ts

```diff
@@ -0,0 +1,90 @@
+import { isString } from "util";
```

Comment:
isString is deprecated, please use: typeof value === 'string' instead. I think in this case we can remove this line because it is not being used anyway.